feat(curriculum): add Movie Ticket Booking Calculator workshop to Python cert (#64866)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-movie-ticket-booking-calculator/6958e1875e592b58e8852757.md

---
id: 6958e1875e592b58e8852757
title: Step 11
challengeType: 20
dashedName: step-11
---

# --description--

Now create a variable named `extra_charges` and set it to `0`. This will represent extra charges to apply to the movie ticket on weekends.

Then, create an `if` statement to check if `is_weekend` is `True`. Inside the body of `if` statement, update the `extra_charges` value to `2` and print `Extra charges will be applied` in the terminal.

# --hints--

You should declare a variable named `extra_charges`.

```js
({ test: () => (runPython(`assert _Node(_code).has_variable("extra_charges")`)) })
```

You should assign the integer value `0` to `extra_charges`.

```js
({ test: () => (runPython(`
assert _Node(_code).find_variable("extra_charges").is_equivalent("extra_charges = 0")
`)) })
```

> You should have an `if` statement.

Reviewer: ```suggestion
You should have a fourth `if` statement.
```

```js
({ test: () => (runPython(`assert _Node(_code).find_ifs()[3]`)) })
```

> The if statement condition should be `is_weekend`.

Reviewer: ```suggestion
Your fourth `if` statement condition should be `is_weekend`.
```

```js
({ test: () => (runPython(`
assert _Node(_code).find_ifs()[3].find_conditions()[0].is_equivalent("is_weekend")
`)) })
```

> You should assign the integer value `2` to `extra_charges` variable inside your `if` statement.

Reviewer: ```suggestion
You should assign the integer value `2` to `extra_charges` variable inside your new `if` statement.
```

```js
({ test: () => (runPython(`
_var = _Node(_code).find_ifs()[3].find_bodies()[0].find_variable("extra_charges") 
assert _var.is_equivalent("extra_charges = 2")
`)) })
```

> You should print `Extra charges will be applied` inside your `if` statement. Remember to surround the message with single or double quotes.

Reviewer: ```suggestion
You should print `Extra charges will be applied` inside your new `if` statement. Remember to surround the message with single or double quotes.
```
